test/e2e/keycloak_test.go

```diff
@@ -63,12 +69,24 @@ func keycloakDeploymentTest(t *testing.T, f *framework.Framework, ctx *framework
 	if err != nil {
 		return err
 	}
-	expectInternalURL := fmt.Sprintf("https://%s.%s.svc:%d",
```

Comment:
This check is no longer necessary. Once we introduce other tests (for all other Pull Requests) we will be validating the `InternalURL` implicitly.